Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM denoland/deno:alpine-1.11.0
+FROM denoland/deno:alpine-1.14.1
```

Comment:
I'll bump the Deno version in a separate PR, please rebase once it's done instead of doing it here